support-workers/build.sbt

```diff
@@ -53,6 +53,7 @@ assemblyJarName := s"${name.value}.jar"
 assemblyMergeStrategy in assembly := {
   case PathList("models", xs@_*) => MergeStrategy.discard
   case x if x.endsWith("io.netty.versions.properties") => MergeStrategy.first
+  case x if x.endsWith("module-info.class") => MergeStrategy.discard
```

Comment:
necessary because of a `deduplicate: different file contents found in the following` error caused by jackson + joda-convert.